app/generated_resources.grd

```diff
@@ -7336,7 +7336,7 @@ Check your passwords anytime in <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Brave P
             Microsoft Internet Explorer
           </message>
           <message name="IDS_IMPORT_FROM_EDGE" desc="browser combo box: Microsoft Edge">
-            Microsoft Edge
+            Microsoft Edge Legacy
```

Comment:
Question for @mkarolin @emerick - will this get overwritten when we run l10n scripts?

Reply:
Yes, indeed. A new string, e.g. `IDS_BRAVE_IMPORT_FROM_EDGE` can be added to `app/brave_generated_resources.grd` and then in `chromium_src/chrome/browser/importer/importer_list.cc` you can do `#define IDS_IMPORT_FROM_EDGE IDS_BRAVE_IMPORT_FROM_EDGE`

Reply:
Perfect! resolved above 👍